### PR TITLE
[INLONG-1029][Doc] Add TubeMQ binary protocol doc

### LINKED ISSUE #1029: [Improve][Doc] Add TubeMQ binary protocol doc
### Description

Add an introduction to the definition of the TubeMQ signaling protocol and how to use the protocol for data production and consumption

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gosonzhang**
Fixes #1029 

1. Remove the original TubeMQ RPC document introduction;
2. Add TubeMQ binary protocol definition and interaction documents

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+            + AddressUtils.getLocalAddress() + "_" // 本机IP (IPV4)
+            + pid + "_" // 进程ID
+            + timestamp + "_" // 时间戳
+            + counter + "_" // 自增计数器
+            + consumerType + "_" // 消费者类型，包含 Pull 和 Push 两种类型
+            + clientVersion; // 客户端版本号
+  ```
+  建议其他语言增加如上标记，以便于问题排查；
+
+- TopicList：标识用户发布的 Topic 列表，Producer 在初始化时候会提供初始的待发布数据的 Topic 列表，在运行中也允许业务通过 Publish 函数延迟的增加新的 Topic 及减少已 Publish 的 topic；
```
Comment (luchunliang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md:188]: TopicList is a parameter, topicList?

**Review by luchunliang — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+- 生产过程中遇到 Master 连接失败时，比如超时，链接被动断开等，Producer 必须发起到 Master 重注册请求；
+
+- Producer 从 Master 收到 Topic 的元数据信息后要提前做到 Broker 预连接操作，避免数据生产时突增大量链接请求影响消息上报性能；
+
+- Producer 到 Broker 的连接要做异常检测：长期运行场景要能检测出 Broker 故障节点，并且对于长期不发消息的链接要将其回收，提高数据上报服务的稳定性。
+
+### Consumer 消费交互图
+
+Consumer 一共使用了 7 对指令：与 Master 进行注册，心跳，注销操作；与 Broker 进行注册，注销，心跳，拉取消息，确认消息操作。其中到 Broker 的注册、注销使用了同一个命令名携带不同的状态码方式来标识和区分不同操作：
+
```
Comment (luchunliang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md:121]: "7对指令" is not clear.

**Review by luchunliang — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md (hunk 1 above)
Comment (gosonzhang) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md:188]: Ok, thanks!

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md (hunk 2 above)
Comment (gosonzhang) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/development/binary_protocol/tubemq_binary.md:121]: You're right!

**Review by aloyszhang — APPROVED** [review]

**Pull request merged**